### ISSUE TITLE
Support mypy

### ISSUE THREAD

**Issue by Paradoxeuh** [atom-community/ide-python#126]
Hello,

I tried recently to install pyls-mypy, and it was not working at all. It broke the package in atom. Is it something you plan to support?

Thanks

**Comment by piox34** [comment]
It would also be great to add support for black and isort, the other two optional extensions. Hopefully this should be straightforward.

**Comment by tillhainbach** [comment]
You can add mypy support manually:
run
```
$ python3 -m pip install git+https://github.com/tomv564/pyls-mypy.git
```

and then add this to `~/.atom/packages/ide-python/package.json`
```json
        "pyls_mypy": {
          "title": "MyPy",
          "type": "object",
          "properties": {
            "enabled": {
              "title": "Enabled",
              "type": "boolean",
              "order": 1,
              "default": true,
              "description": "Enable or disable MyPy."
            },
            "live_mode": {
              "title": "Live mode",
              "type": "boolean",
              "order": 1,
              "default": false,
              "description": "Turn on live mode (If disabled, mypy linters on file save)."
            }
          }
        }
```

Use Toggles under MyPy to enable/disable mypy warnings and switch between live-mode and "on file save"

**Comment by aminya** [comment]
Already fixed in #232
